### ISSUE TITLE
train model error

### ISSUE THREAD

**Issue by 3477026455** [zcemycl/TF2DeepFloorplan#84]
use command "python -m src.dfp.train --batchsize=4 --lr=5e-4 --epochs=3000 --logdir=log/store --modeldir=model/store" to train a model,when I use the command "python -m src.dfp.deploy --image resources/123.jpg --weight log/store/G --postprocess --colorize --save output.jpg --loadmethod log", the output image only display the segmentation of wall 、window and door.why is that ?

**Comment by 3477026455** [comment]
![output](https://github.com/user-attachments/assets/a060e0d5-b7b3-472d-adc0-c5c4acdf1eb8)
output image like this

**Comment by zcemycl** [comment]
@3477026455 it is a gradient collapse due to overfitting. I usually only train on less than 500 epochs, or slower learning rate, unless you have many training data. 
(An observation i had few years ago, if the room-boundary attention is learnt too quickly, the room type detection which depends on it can lead to gradient collapse showing all black)
